lib/ansible/parsing/vault/__init__.py

```diff
@@ -357,943 +65,17 @@ def get_file_vault_secret(filename=None, vault_id=None, encoding=None, loader=No
     # while files from --vault-password-file are already unfracked, other sources are not
     this_path = unfrackpath(filename, follow=False)
     if not os.path.exists(this_path):
-        raise AnsibleError("The vault password file %s was not found" % this_path)
+        raise AnsibleError(f"The vault password file {this_path} was not found")
 
     # it is a script?
     if loader.is_executable(this_path):
 
-        if script_is_client(filename):
+        if file.script_is_client(filename):
             # this is special script type that handles vault ids
-            display.vvvv(u'The vault password file %s is a client script.' % to_text(this_path))
-            # TODO: pass vault_id_name to script via cli
```

Comment:
Please leave the TODO.

Reply:
You're right, thanks for spotting that. The commit that added `_build_command` moved around that comment, but forgot to remove it I think.